docs/docsite/rst/community/committer_guidelines.rst

```diff
@@ -4,50 +4,58 @@
 Committers Guidelines
 *********************
 
-These are the guidelines for people with commit privileges on the Ansible GitHub repository. Committers are essentially acting as members of the Ansible Core team, although not necessarily as employees of Ansible and Red Hat. Please read the guidelines before you commit.
+These are the guidelines for people with commit privileges on the Ansible GitHub repositories.
```

Comment:
This includes collections repos, right? How about:
```suggestion
These are the guidelines for people with commit privileges on the repositories in the ansible and ansible-collections GitHub organizations.
```

Reply:
sounds good!